scheduler/test/cook/test/tools.clj

```diff
@@ -20,9 +20,11 @@
             [clojure.core.cache :as cache]
             [clojure.test :refer :all]
             [cook.config :as config]
+            [cook.queries :as queries]
             [cook.test.testutil :as testutil
              :refer [create-dummy-group create-dummy-instance create-dummy-job create-dummy-job-with-instances create-pool restore-fresh-database!]]
-            [cook.tools :as util]
+            [cook.tools :as tools]
```

Comment:
I feel the refactor of util to tools can also be split off.

Reply:
Discussed offline the pros/cons of splitting this up, I decided to keep it as one PR